### DISCUSSION TITLE
Quiz and Assignments

### DISCUSSION THREAD

**Discussion by amrshaikh** [microsoft/Web-Dev-For-Beginners#1331] [Q&A]
Hey everyone!

I've got two questions.
How do I take the quizzes?
How do I submit the assignments?

Thanks.

**Comment by Cool-boy0** [comment]
Taking Quizzes
1. Clone the repository: `git clone <repository_url>`
2. Complete the quiz: Follow the instructions.
3. Commit and push: 
   bash
   git add .
   git commit -m "Completed quiz"
   git push origin main
   

      Submitting Assignments
1. Clone the repository: `git clone <repository_url>`
2. Complete the assignment: Follow the instructions.
3. **Commit and push**: 
   bash
   git add .
   git commit -m "Submitting assignment"
   git push origin main
   
https://www.youtube.com/watch?v=--x1yIj44-o

**Reply by Cool-boy0** [reply]
i hope it is helpfull

**Reply by amrshaikh** [reply]
thanks! i'll check it out

**Comment by Pankajdas0025** [comment]
**1️⃣ Taking the quizzes**
Most quiz links are shared inside the repository’s README or in the specific module folder.
Just open the quiz link, complete all questions, and submit — your responses will be saved automatically.

**2️⃣ Submitting the assignments**
Each assignment usually has its own folder.
Fork the repo → complete the task in your fork → push your solution → then create a Pull Request back to the main repository.
This lets the maintainers review your work and mark it as submitted.
If the project follows a different method, the maintainers normally mention it in the instructions file for that module.
Hope this helps!

**Reply by joanaquaicoe2-cmyk** [reply]
I wish I knew something I would have liked to be very much helpful too please I need help 🙏🥺
